code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -69,7 +70,7 @@
 		var/mob/living/carbon/xenomorph/xeno = locate(href_list["hivemind_jump"])
 		if(!istype(xeno))
 			return
-		var/obj/effect/alien/weeds/nearby_weed = locate() in range("3x3", get_turf(xeno))
+		var/obj/effect/alien/weeds/nearby_weed = locate() in range("6x6", get_turf(xeno)) //increased
```

Comment:
Wasa there any reason this was changed in this PR? This is unrelated to this PR.

Reply:
@psykzz yes actually, the range you could find xenos in (jumping to) was unreliable in a 3x3 zone, even though you could then jump to them if you could manually go to weeds, and then doubleclick them